src/components/EzField/EzField.tsx

```diff
@@ -192,6 +201,8 @@ const EzField = forwardRef<HTMLElement, Props>((props, ref) => {
                 id,
                 name: props.name || id,
                 'aria-labelledby': labelId,
+                'aria-invalid': showError,
+                'aria-describedby': clsx(helperTextId || labelId, showError && errorMessageId),
```

Comment:
This doesn't look quite right, but perhaps I'm missing something.

From what I can tell, `helperTextId` always has a value (`helperText-${id}`), so `labelId` will never be applied. I *think* that's probably OK, since `labelId` probably _shouldn't_ be applied to `aria-describedby`, since it is already used for `aria-labelledby`. That being said, code paths that aren't used (and we don't want to be used) should probably be removed.

It might also make sense to lead with the error message, since it provides the most relevant contextual information first.

Perhaps we want something like:
```suggestion
                'aria-describedby': clsx(showError && errorMessageId, helperText && helperTextId),
```

Reply:
updated